siriushlacon/countingpru/main.py

```diff
@@ -43,14 +43,13 @@ def __init__(self, parent=None, macros=None, args=None):
 
         self.counterBox.currentIndexChanged.connect(partial(self.new_counter, True))                            #True: chosen on QComboBox, False: Wrote by user
         self.gammaBox.currentIndexChanged.connect(partial(self.new_detector, True))                             # ""
-        self.Counter_name.editingFinished.connect(self.new_counter)                                             #Set function for Counter and Gamma sensor
-        self.Gamma_name.editingFinished.connect(self.new_detector)
+        self.Counter_name.returnPressed.connect(self.new_counter)                                               #Set function for Counter and Gamma sensor
+        self.Gamma_name.returnPressed.connect(self.new_detector, True)
 
-        self.Plot_Button.clicked.connect(self.plot_info)                                                        #
         self.BT_Button.clicked.connect(self.Set_TimeBase)                                                       #
         self.OverviewButton.clicked.connect(self.overview)                                                      #Set functions to bottons
         self.pulsesAverageButton.clicked.connect(self.plot_average)                                             #
-        self.OverviewAverage.clicked.connect(partial(self.plot_average, True))                                  #
+        self.OverviewAverage.clicked.connect(partial(self.plot_average, True, True))                            #
 
     def plot_average(self, overview = False):
```

Comment:
Quando clico nesse botão, retorna um erro: _"TypeError: plot_average() takes from 1 to 2 positional arguments but 4 were given"_